<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadWindow</class>
 <widget class="QMainWindow" name="UploadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="UploadButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bungee Inline</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QLabel" name="NameLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bungee Inline</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NAME:SAMPLETEXT</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="RefreshButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bungee Inline</family>
     </font>
    </property>
    <property name="text">
     <string>Refresh fie list</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>421</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1304</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
